highlight.io/components/QuickstartContent/traces/node-js/nextjs.tsx

```diff
@@ -0,0 +1,81 @@
+import { siteUrl } from '../../../../utils/urls'
+import { QuickStartContent } from '../../QuickstartContent'
+import { verifyTraces } from '../shared-snippets'
+
+export const NextJsTracesContent: QuickStartContent = {
+	title: 'Next.js',
+	subtitle:
+		'Learn how to set up highlight.io tracing for your Next.js application.',
+	logoUrl: siteUrl('/images/quickstart/nextjs.svg'),
+	entries: [
+		{
+			title: 'Install the Next.js SDK',
+			content: 'Install the Next.js SDK using your package manger.',
+			code: [
+				{
+					text: `npm install --save @highlight-run/next`,
```

Comment:
we have a helper for this snippet
https://github.com/highlight/highlight/blob/main/highlight.io/components/QuickstartContent/backend/js/shared-snippets.tsx#L4-L36

Reply:
How do you feel about removing references to `yarn` and `pnpm` site-wide?

We removed them on the Next.js docs, and I kinda like it? Anyone using `yarn`, `pnpm` or `bun` knows the correct commands, so it tightens up our docs.

![image](https://github.com/highlight/highlight/assets/878947/6d5b6220-41d3-459c-8eed-e3d5c36eac4c)